src/core/loader/get-catalogue.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { initCatalogue } from "../../components/shop/item-catalogue.js";
+
+export const loadCatalogue = (screen, config) => {
+    const catalogueKeys = getCatalogueKeys(config);
+
+    catalogueKeys.forEach(loadToCache(screen));
+
+    screen.load.on("complete", () => {
```

Comment:
this only needs to be called once I guess, so you can use `.once` instead

Reply:
could probably get rid of some of the `{ }` in this callback too

Reply:
f84677e